src/minimizer_mapper.cpp

```diff
@@ -1404,14 +1288,13 @@ pair<vector<Alignment>, vector< Alignment>> MinimizerMapper::map_paired(Alignmen
     paired_scores.reserve(alignments.size());
     vector<int64_t> fragment_distances;
     fragment_distances.reserve(alignments.size());
+    
+    vector<size_t> better_cluster_count_by_pairs;
```

Comment:
This should probably have a comment explaining what it holds. I guess it's, for each pair, the number of better clusters?